infra/base-images/base-builder/detect_repo.py

```diff
@@ -29,26 +31,35 @@
 
 
 def main():
-  """Function to get a git repos information based on its commit."""
+  """Function to get a git repo's url and name refenced by OSS-Fuzz Dockerfile.
+
+  Raises:
+    ValueError when a commit or a ref is not provided.
+  """
   parser = argparse.ArgumentParser(
-      description='Finds a specific git repo in an oss-fuzz projects docker file.'
-  )
+      description=
+      "Finds a specific git repo in an oss-fuzz project's docker file.")
```

Comment:
nit: use single quotes for such strings

Reply:
There is an apostrophe in the description.

Reply:
Doesn't matter. Escape the apostrophe then.

Reply:
Done.